### ISSUE TITLE
Dependabot does not keep package up to date.

### ISSUE THREAD

**Issue by jkoelker** [plaid/plaid-go#235]
Due to the reversioning of the package back down to `v0.0.0` and from there following semver, after publishing a non-go module tag with the version string of `7.0.0`, dependabot fails to update this package when new versions are released. Would it be possible to release the next version as `v8.0.0` to allow dependabot (and other tools that use tags for version comparison) to detect that the library has been updated?

**Comment by jkoelker** [comment]
@ghernandez-plaid Yes our product is based on the plaid data and we are actively using it. Shoot the link over to the email on my profile and I'll FWD it over to my company account and schedule it from there.

**Comment by ghernandez-plaid** [comment]
> @ghernandez-plaid Yes our product is based on the plaid data and we are actively using it. Shoot the link over to the email on my profile and I'll FWD it over to my company account and schedule it from there.

Awesome! I just sent over an email to the email on your profile. Looking forward to chatting! Thank you.

**Comment by phoenixy1** [comment]
@jkoelker we're doing the next release on tuesday and will plan to bump the version then

**Comment by ghernandez-plaid** [comment]
Hi @jkoelker, just wanted to follow up here on your original issue. In the latest release for this library we bumped the version back to 8.0.0. We're hoping this change will make automatic version updates easier! For more context, see the description on this pull request: https://github.com/plaid/plaid-go/pull/245

I'm going to close Issue, but free to re-open in case other issues arise. Thanks!

- Gilberto from DevRel
